tortoise-hg.sls

```diff
@@ -1,34 +1,16 @@
-# Source: http://tortoisehg.bitbucket.org/
+{% if grains['cpuarch'] == 'AMD64' %}
+{% set msi_arch = 'x64' %}
+{% else %}
+{% set msi_arch = 'x86' %}
+{% endif %}
+
 tortoise-hg:
```

Comment:
We still need to keep the older versions around so that they can be uninstalled.

Reply:
Didn't think of that, of course, makes sense. I pushed a correction.

Sadly the old 3.3.0 and 3.6.2 are not currently available from the official download site, so we have to hope that people that kept the MSIs.

I've asked the TortoiseHg maintainers.